lib/assets/models/index.js

```diff
@@ -14,15 +14,26 @@ if (config.use_env_variable) {
   var sequelize = new Sequelize(config.database, config.username, config.password, config);
 }
 
-fs
-  .readdirSync(__dirname)
-  .filter(function(file) {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
-  })
-  .forEach(function(file) {
-    var model = sequelize['import'](path.join(__dirname, file));
-    db[model.name] = model;
-  });
+function readModels(dir) {
+  if(!fs.lstatsync(dir).isDirectory()) {
+    return;
+  }
+    fs.readdirSync(dir)
+      .filter(function (file) {
+        return file !== basename;
+      })
+      .forEach(function (file) {
+        var tmpPath = path.join(dir, file);
+        if ((!fs.lstatSync(tmpPath).isDirectory()) && (tmpPath.slice(-3) === '.js')) {
```

Comment:
these conditions should be separated, you don't want it to call readModels if the file isn't ending with '.js'

You also removed the condition that does not read hidden files.